stock_available_to_promise_release/readme/DESCRIPTION.rst

```diff
@@ -32,3 +32,8 @@ the remaining qty is split and attached to a new picking that can be released la
 
 Important: if the "Stock reservation horizon" is set, the qty is calculated
 on moves which have an expected date not beyond $today + $horizon (in days).
+
+Important: Moves without a warhouse set are not supported.
+If the warehouse is not set on the move, the "Ordered Available to Promise" Qty 
+will always be 0. The warehouse gets set by the Stock rule, if the the Rule
+is not assigned to a warehouse, also the move will not end up with a warehouse.
```

Comment:
```suggestion
will always be 0. It will also not be considered as consuming existing stock.
The warehouse gets set by the Stock rule, if the the Rule
is not assigned to a warehouse, also the move will not end up with a warehouse.
```